test/config.py

```diff
@@ -0,0 +1,6 @@
+"""
+This file is used to configure any shared or global parameters for the test
+suite.
+"""
+
+CORRECTNESS_SEED = 0
```

Comment:
Can we add other things like the datasets directory?

Reply:
Yes, that is exactly what this can be used for. This file seems like a reasonable solution to single source shared variables. If you know of another way, I am not set on this.